juniper_official/System/system-storage.rule

```diff
@@ -169,6 +169,10 @@ healthbot {
             rule-properties {
                 version 2;
                 contributor juniper;
+                category basic;
+                is-scaling-rule {
```

Comment:
Not scale rule as juniper device have less mount paths

Reply:
Removed scaling.